doc/nrf/ug_tfm.rst

```diff
@@ -7,7 +7,7 @@ Running applications with Trusted Firmware-M
    :local:
    :depth: 2
 
-On nRF5340 and nRF9160, you can use Trusted Firmware-M (TF-M) as an alternative to :ref:`secure_partition_manager` for running an application from the non-secure area of the memory.
+On nRF5340 and nRF9160, Trusted Firmware-M (TF-M) is used for running an application in the non-secure state.
```

Comment:
I have some problem with this statement. TF-M is implementing the secure image and has no knowledge about what the application is doing

How about:

> On nRF5340 and nRF9160, Trusted Firmware-M (TF-M) can be used to establish root-of-trust when running an appliation in the non-secure state.

Reply:
What this sentence is saying is that TF-M is responsible for configuring the application to run in the non-secure state. The application does not put itself as non-secure.

Reply:
```suggestion
On nRF5340 and nRF9160, Trusted Firmware-M (TF-M) can be used to configure and boot an application as non-secure.
```